src/content/docs/en/pages/guides/account-lockout-policy/configure-account-lockout-policy.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: How to configure Account Lockout Policy
+description: This guide walks you through configuring the Account Lockout Policy feature to protect your account from unauthorized access. 
+meta_tags: account management, security, brute force attacks, Azion Account Lockout Policy
+namespace: docs_guides_account_lockout_policy
+permalink: /documentation/products/guides/configure-account-lockout-policy/
+---
+
+import LinkButton from 'azion-webkit/linkbutton'
+
+The **Account Lockout Policy** feature allows you to define the maximum number of failed attempts a user can complete before blocking the access, for a predefined period. During this period, the user can't attempt to log in again, preventing unauthorized access.
+
+This guide walks you through configuring the Account Lockout Policy feature.
+
+<LinkButton link="/en/documentation/products/accounts/account-lockout-policy/" label="Go to Account Lockout Policy reference" severity="secondary" target="_blank" />
+
+:::note
+When an organization sets the Account Lockout Policy up, the policy applies to all account levels.
```

Comment:
When an organization implements the Account Lockout Policy, it applies to all account levels.